src/main/resources/hudson/plugins/claim/AbstractClaimBuildAction/tablerow.jelly

```diff
@@ -1,82 +1,82 @@
 <?jelly escape-by-default='true'?>
-<td>
-    <?jelly escape-by-default='true'?>
-    <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <j:if test="${it.isColumnDisplayed()}">
+        <td class="pane">
+            <script type="text/javascript">
+                function ShowPopup(hoveritem) {
+                var hp = document.getElementById("claimHoverPopup");
+                hp.style.display = "block";
+                }
 
-        <script type="text/javascript">
-            function ShowPopup(hoveritem) {
-            var hp = document.getElementById("claimHoverPopup");
-            hp.style.display = "block";
-            }
+                function HidePopup() {
+                var hp = document.getElementById("claimHoverPopup");
+                hp.style.display = "none";
+                var action = <st:bind value="${it}" />;
+                }
 
-            function HidePopup() {
-            var hp = document.getElementById("claimHoverPopup");
-            hp.style.display = "none";
-            var action = <st:bind value="${it}" />;
-            }
+                function Display(error) {
+                var reasonText = document.getElementById("errordesc");
+                var action = <st:bind value="${it}" />;
+                action.getReason(error, function(content) {
+                reasonText.textContent = content.responseObject();
+                });
+                }
+            </script>
 
-            function Display(error) {
-            var reasonText = document.getElementById("errordesc");
-            var action = <st:bind value="${it}" />;
-            action.getReason(error, function(content) {
-            reasonText.textContent = content.responseObject();
-            });
-            }
-        </script>
+            <j:set var="linkWritten" value="false"/>
+            <j:if test="${it.canClaim()}">
+                <j:if test="${linkWritten}"><st:nbsp/></j:if>
+                <a id="claim" href="#" onClick="ShowPopup(this); return false;">
+                    <j:out value="${it.messageProvider.getClaimActionText()}"/>
+                </a>
+                <j:set var="linkWritten" value="true"/>
+            </j:if>
+            <j:if test="${it.canRelease()}">
+                <j:if test="${linkWritten}"><st:nbsp/></j:if>
+                <f:link href="claim/unclaim" post="true">
+                    <j:out value="${it.messageProvider.getUnclaimActionText()}"/>
+                </f:link>
+                <j:set var="linkWritten" value="true"/>
+            </j:if>
+            <j:if test="${it.canReassign()}">
+                <j:if test="${linkWritten}"><st:nbsp/></j:if>
+                <a id="reassign" href="#" onClick="ShowPopup(this); return false;">
```

Comment:
I would add an \<br/> tag here for better formatting.
![image](https://github.com/jenkinsci/claim-plugin/assets/57817042/13020a39-0a70-4e56-89f7-bdb831270cc1)
![image](https://github.com/jenkinsci/claim-plugin/assets/57817042/58cb2136-ec19-40d9-b410-68cad546e361)